.github/workflows/ci.yml

```diff
@@ -8,11 +8,11 @@ concurrency:
 
 jobs:
   tests:
-    continue-on-error: true
     strategy:
+      fail-fast: false
```

Comment:
Disable `fail-fast`. This prevents GitHub Actions from canceling remaining jobs if any job in the matrix fails.